styles.css

```diff
@@ -33,4 +33,18 @@ body {
 
 .block--gray {
   background-color: gray;
-}
\ No newline at end of file
+}
+
+.go-right {
```

Comment:
Nice thought process here!! It's pretty unique that you made the function call here!